frontend/tests/unit/store/tagStore.spec.ts

```diff
@@ -0,0 +1,118 @@
+import { setActivePinia, createPinia } from 'pinia';
+
+import * as primevue from '@/lib/primevue';
+import { objectService } from '@/services';
+import { useAppStore, useTagStore } from '@/store';
+
+import type { Tagging } from '@/types';
+
+const mockAdd = vi.fn();
+const useToastSpy = vi.spyOn(primevue, 'useToast').mockImplementation(() => ({ add: mockAdd }));
+
+beforeEach(() => {
+  setActivePinia(createPinia());
+  vi.clearAllMocks();
+});
+
+afterEach(() => {
+  sessionStorage.clear();
+});
+
+describe('Config Store', () => {
+
+  describe('fetchTagging', () => {
+    it('fetches the tags', async () => {
+      const appStore = useAppStore();
+      const tagStore = useTagStore();
+
+      const tag: Tagging = {
```

Comment:
As 'tag' variable has been used 4 times in the test, it might be better for 'tag' to be defined at the top level (line 11 for example) rather than to be function nested.